clients/ui/bff/internal/api/middleware.go

```diff
@@ -109,9 +109,8 @@ func (app *App) AttachModelCatalogRESTClient(next func(http.ResponseWriter, *htt
 		modelCatalogBaseURL := modelCatalog.ServerAddress
 
 		// If we are in dev mode, we need to resolve the server address to the local host
-		// to allow the client to connect to the model registry via port forwarded from the cluster to the local machine.
-		// If you are in federated mode, we do not want to override the server address.
-		if app.config.DevMode && !app.config.DeploymentMode.IsFederatedMode() {
+		// to allow the client to connect to the model catalog via port forwarded from the cluster to the local machine.
+		if app.config.DevMode {
```

Comment:
@lucferbux do you recall why you added that?

@mturley if we should remove that, could you please also remove it from MR?

Reply:
I think we still want it for MR? not sure. we're still using the external route for MR for now and it works 😛